### DISCUSSION TITLE
Проверка письма из Cypress

### DISCUSSION THREAD

**Discussion by Alex-QA-Test** [open-tutorials/cypress#76] [Q&A]
Как лучше реализовать данный текст кейс?

- Заходим на страницу регистрации
- Заполняем нужные поля, нажимаем кнопку "Далее"
- Переходим на этап ввода пароля
У нас улетает запрос с заполненными данными и письмо с паролем. Идем например на MailHog, находим наше письмо, достаем из него пароль.
- Возвращаемся на этап ввода пароля и вводим пароль.

**Comment by Vladimir-Tarasov** [comment]
Добрый вечер. Не понятна суть вопроса.

**Comment by breslavsky** [comment]
По сути это регистрация с подтверждением по email, верно? Я добавлю этот кейс ко вторнику сюда https://www.epic1h.com/fun_cases_in_cypress

**Reply by Alex-QA-Test** [reply]
Да, всё верно

**Reply by breslavsky** [reply]
Как и обещал [туториал](https://www.epic1h.com/fun_cases_in_cypress#5-%D1%80%D0%B5%D0%B3%D0%B8%D1%81%D1%82%D1%80%D0%B0%D1%86%D0%B8%D1%8F-%D1%81-%D0%BF%D0%BE%D0%B4%D1%82%D0%B2%D0%B5%D1%80%D0%B6%D0%B4%D0%B5%D0%BD%D0%B8%D0%B5%D0%BC-email)
